AKS-Hybrid/deploy-gpu-node-pool.md

```diff
@@ -79,10 +104,21 @@ Error       3D Video Controller                   PCI\VEN_10DE&DEV_1EB8&SUBSYS_1
 
 When you uninstall the host driver, the physical GPU goes into an error state. You must dismount all the GPU devices from the host.
 
-For each GPU (3D Video Controller) device, run the following commands in PowerShell. Copy the instance ID; for example, `PCI\VEN_10DE&DEV_1EB8&SUBSYS_12A210DE&REV_A1\4&32EEF88F&0&0000` from the previous command output:
+For each GPU (3D Video Controller) device, run the following commands in PowerShell. This command will create a variable named "ID1" and "lp1" and populate the instance ID of the first GPU; for example, `PCI\VEN_10DE&DEV_1EB8&SUBSYS_12A210DE&REV_A1\4&32EEF88F&0&0000` from the previous command output. 
+
+```powershell
+$gpu=Get-PnpDevice -FriendlyName "3D Video Controller"
+$id0 =$gpu[0].InstanceId
+$lp0 = (Get-PnpDeviceProperty -KeyName DEVPKEY_Device_LocationPaths -InstanceId $id1).Data[0]
```

Comment:
You renamed the variables to `$id0` and `$lp0` here, but subsequence lines (112-114) are still using `$id1` and `$lp1`.

Reply:
i see the issue, quick fix.